src/test/java/core/basesyntax/fruitoperation/OperationsTest.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.fruitoperation;
+
+import junit.framework.TestCase;
+
+public class OperationsTest extends TestCase {
```

Comment:
can we use @Test annotation above each test?

Reply:
done